source/docs/FlytOS/GettingStarted/ReadytouseApps/VisualFollowMeApp.rst

```diff
@@ -50,11 +50,11 @@ Setting up and deep dive:
 		$ roslaunch vision_apps cam_api.launch
 
 
-* In case you want to change the camera capture settings, edit following launch file -
+* In case you want to change the camera capture settings, edit the following launch file -
 
-.. code-block:: bash
 
-		/flyt/flytos/flytcore/share/vision_apps/launch/cam_api.launch
+     **/flyt/flytos/flytcore/share/vision_apps/launch/cam_api.launch**
```

Comment:
did you verify **/flyt/flytos/flytcore/share/vision_apps/launch/cam_api.launch** renders correctly?